docs/src/main/paradox/reference.md

```diff
@@ -236,10 +233,35 @@ Generate a `tile` with the values from `data_tile`, with NoData in cells where t
 ### rf_mask_by_values
 
     Tile rf_mask_by_values(Tile data_tile, Tile mask_tile, Array mask_values)
-    Tile rf_mask_by_values(Tile data_tile, Tile mask_tile, seq mask_values)
+    Tile rf_mask_by_values(Tile data_tile, Tile mask_tile, list mask_values)
 
 Generate a `tile` with the values from `data_tile`, with NoData in cells where the `mask_tile` is in the `mask_values` Array or list. `mask_values` can be a [`pyspark.sql.ArrayType`][Array] or a `list`.  
 
+### rf_mask_by_bit
+
+    Tile rf_mask_by_bits(Tile tile, Tile mask_tile, Int bit_position, Bool mask_value) 
+    
+Applies a mask using bit values in the `mask_tile`. Working from the right, the bit at `bit_position` is @ref:[extracted](reference.md#rf_local_extract_bits) from cell values of the `mask_tile`. In all locations where these are equal to the `mask_value`, the returned tile is set to NoData; otherwise the original `tile` cell value is returned.
+
+This is a single-bit version of @ref:[`rf_mask_by_bits`](reference.md#rf-mask-by-bits).
+    
+### rf_mask_by_bits
+
+    Tile rf_mask_by_bits(Tile tile, Tile mask_tile, Int start_bit, Int num_bits, Array mask_values) 
+    Tile rf_mask_by_bits(Tile tile, Tile mask_tile, Int start_bit, Int num_bits, list mask_values) 
+    
+Applies a mask from blacklisted bit values in the `mask_tile`. Working from the right, the bits from `start_bit` to `start_bit + num_bits` are @ref:[extracted](reference.md#rf_local_extract_bits) from cell values of the `mask_tile`. In all locations where these are in the `mask_values`, the returned tile is set to NoData; otherwise the original `tile` cell value is returned.
+
+This function is not available in the SQL API. The below is equivalent:
```

Comment:
You'll have to explain to me why... If it's array creation, it should be available with the `array(x, y, z, ...)` function.

Reply:
Basically I threw in the towel on trying to create an Expression with 5 arguments (tile, tile, int, int, array) ... If you have any intiution on this it's welcome! Maybe we can file another issue to bring that along.